charmhelpers/contrib/network/ovs/__init__.py

```diff
@@ -92,8 +93,20 @@ def add_ovsbridge_linuxbridge(name, bridge):
             apt_install('python3-netifaces', fatal=True)
         import netifaces
 
-    ovsbridge_port = "veth-" + name
-    linuxbridge_port = "veth-" + bridge
+    if len(bridge) <= 10 and len(name) <= 10:
```

Comment:
nit:
MAX_KERNEL_INTERFACE_NAME_LEN = 15
VETH_PREFIX = 'veth-'
BRIDGE_NAME_LEN_LIMIT = MAX_KERNEL_INTERFACE_NAME_LEN - len(VETH_PREFIX)
if len(bridge) <= BRIDGE_NAME_LEN_LIMIT and len(name) <= BRIDGE_NAME_LEN_LIMIT: